From: Felipe Borges <felipeborges@gnome.org>
Date: Tue, 14 Oct 2025 10:23:03 +0200
Subject: housekeeping: Downgrade donation reminder urgency to "normal"

Critical notifications can bypass the do-not-disturbe settings and
interrupt fullscreen video/presentations.

We don't want our donation campaigns to disturb users in focus moments.

Reported in https://gitlab.gnome.org/GNOME/gnome-settings-daemon/-/issues/896#note_2575412

Origin: upstream, 49.2, commit:54b5d05e89873158b20fe83a1f7f7bd44bc10e72
---
 plugins/housekeeping/gsd-donation-reminder.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/housekeeping/gsd-donation-reminder.c b/plugins/housekeeping/gsd-donation-reminder.c
index d5e3f5d..f8adf14 100644
--- a/plugins/housekeeping/gsd-donation-reminder.c
+++ b/plugins/housekeeping/gsd-donation-reminder.c
@@ -80,7 +80,7 @@ show_notification (void)
 
         notify_notification_set_app_name (notification, _("GNOME"));
         notify_notification_set_hint (notification, "transient", g_variant_new_boolean (TRUE));
-        notify_notification_set_urgency (notification, NOTIFY_URGENCY_CRITICAL);
+        notify_notification_set_urgency (notification, NOTIFY_URGENCY_NORMAL);
         notify_notification_set_timeout (notification, NOTIFY_EXPIRES_DEFAULT);
         notify_notification_set_hint_string (notification, "desktop-entry", "gnome-about-panel");
 
